vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -1094,6 +1095,10 @@ std::vector<std::unique_ptr<ExclusionRule>>
       std::make_unique<TotalMaxFrequencyCap>(frequency_capping_.get());
   exclusion_rules.push_back(std::move(total_max_frequency_cap));
 
+  std::unique_ptr<ExclusionRule> conversion_frequency_cap =
+      std::make_unique<ConversionFrequencyCap>(frequency_capping_.get());
```

Comment:
`frequency_capping_` null check is missing

Reply:
This has been fixed as part of a wider refactor of unit tests/mocks